### PR TITLE
[spec/interpreter/tests] Rename instructions

### PR DESCRIPTION
**Pull request by rossberg**
Implement instruction renamings from [this list](https://github.com/WebAssembly/spec/issues/884#issuecomment-426433329) on #884, as decided at [2018/10/02 CG meeting](https://github.com/WebAssembly/meetings/blob/e43a829840fed8fe17c1061decdc3260d6323e04/2018/CG-10-02.md).

### CODE REVIEW

File: test/core/align.wast [hunk 1]
```diff
@@ -462,178 +462,178 @@
...
-      (set_local 2 (f64.load align=8 (i32.const 0)))
+      (f64.store align=8 (i32.const 0) (local.get 1))
+      (local.set 2 (f64.load align=8 (i32.const 0)))
     ) ;; 8
-    (get_local 2)
+    (local.get 2)
   )
 
-  ;; $8s: i32/i64.load8_s, $8u: i32/i64.load8_u, $16s: i32/i64.load16_s, $16u: i32/i64.load16_u, $32: i32.load
+  ;; $8s: i32_i64.load8_s, $8u: i32_i64.load8_u, $16s: i32_i64.load16_s, $16u: i32_i64.load16_u, $32: i32.load
```
Comment (binji) [inline test/core/align.wast]: bad search/replace

File: test/core/conversions.wast [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-  (func (export "f32.convert_u_i64") (param $x i64) (result f32) (f32.convert_u/i64 (get_local $x)))
-  (func (export "f64.convert_u_i32") (param $x i32) (result f64) (f64.convert_u/i32 (get_local $x)))
-  (func (export "f64.convert_u_i64") (param $x i64) (result f64) (f64.convert_u/i64 (get_local $x)))
-  (func (export "f64.promote_f32") (param $x f32) (result f64) (f64.promote/f32 (get_local $x)))
-  (func (export "f32.demote_f64") (param $x f64) (result f32) (f32.demote/f64 (get_local $x)))
-  (func (export "f32.reinterpret_i32") (param $x i32) (result f32) (f32.reinterpret/i32 (get_local $x)))
-  (func (export "f64.reinterpret_i64") (param $x i64) (result f64) (f64.reinterpret/i64 (get_local $x)))
-  (func (export "i32.reinterpret_f32") (param $x f32) (result i32) (i32.reinterpret/f32 (get_local $x)))
-  (func (export "i64.reinterpret_f64") (param $x f64) (result i64) (i64.reinterpret/f64 (get_local $x)))
+  (func (export "i64.extend_s_i32") (param $x i32) (result i64) (i64.extend_i32_s (local.get $x)))
```
Comment (binji) [inline test/core/conversions.wast]: rename the exports too?

File: test/core/float_exprs.wast [hunk 3]
```diff
@@ -489,17 +489,17 @@
 (assert_return (invoke "f32.no_approximate_sqrt_reciprocal" (f32.const 0x1.ba4c5p+13)) (f32.const 0x1.136f16p-7))
 (assert_return (invoke "f32.no_approximate_sqrt_reciprocal" (f32.const 0x1.4a5be2p+104)) (f32.const 0x1.c2b5bp-53))
 
-;; Test that converting i32/i64 to f32/f64 and back isn't folded away.
+;; Test that converting i32_i64 to f32_f64 and back isn't folded away.
```
Comment (binji) [inline test/core/float_exprs.wast]: here too

File: document/core/appendix/index-instructions.rst [hunk 4]
```diff
@@ -39,11 +39,11 @@ Instruction                          Binary Opcode     Type
...
-:math:`\SETGLOBAL~x`                 :math:`\hex{24}`  :math:`[t] \to []`                          :ref:`validation <valid-set_global>`      :ref:`execution <exec-set_global>`
+:math:`\LOCALGET~x`                  :math:`\hex{20}`  :math:`[] \to [t]`                          :ref:`validation <valid-local.get>`       :ref:`execution <exec-local.get>`
+:math:`\LOCALSET~x`                  :math:`\hex{21}`  :math:`[t] \to []`                          :ref:`validation <valid-local.set>`       :ref:`execution <exec-local.set>`
+:math:`\LOCALTEE~x`                  :math:`\hex{22}`  :math:`[t] \to [t]`                         :ref:`validation <valid-local.tee>`       :ref:`execution <exec-local.tee>`
+:math:`\GLOBALGET~x`                 :math:`\hex{23}`  :math:`[] \to [t]`                          :ref:`validation <valid-global.get>`      :ref:`execution <exec-global.get>`
+:math:`\GLOBALSET~x`                 :math:`\hex{24}`  :math:`[t] \to []`                          :ref:`validation <valid-global.set>`      :ref:`execution <exec-global.set>`
```
Comment (binji) [inline document/core/appendix/index-instructions.rst]: update conversions in this file too

Comment (rossberg) [reply document/core/appendix/index-instructions.rst]: Done.

File: test/core/conversions.wast (hunk 2 above)
Comment (rossberg) [reply test/core/conversions.wast]: Done.

File: test/core/float_exprs.wast (hunk 3 above)
Comment (rossberg) [reply test/core/float_exprs.wast]: Done.

File: test/core/align.wast (hunk 1 above)
Comment (rossberg) [reply test/core/align.wast]: Done.

File: document/core/appendix/index-instructions.rst (hunk 4 above)
Comment (rossberg) [reply document/core/appendix/index-instructions.rst]: Done.

**Pull request merged**